juriscraper/opinions/united_states/territories/nmariana.py

```diff
@@ -60,8 +60,8 @@ def _process_html(self):
                     "name": cells[0].text_content(),
                     "citation": cells[1].text_content(),
                     "date": cells[2].text_content(),
-                    "judge": self._cleanup_judge_names(judge_text),
                     "author": self._fetch_author(judge_text),
+                    "judge": self._fetch_author(judge_text),
```

Comment:
Shouldn't this be:
```python
 "judge": ", ".join(self._cleanup_judge_names(judge_text))
 "author": self._fetch_author(judge_text),
```
Since the info held by "judge" and "author" is different?

The difference between `OpinionCluster.judges` and `Opinion.author_str`

Which in the end is kinda of mixed up in Courtlistener, but that's another issue
https://github.com/freelawproject/courtlistener/blob/7b87ce181eddc389dbefeacff5e270591b99f501/cl/scrapers/management/commands/cl_scrape_opinions.py#L168-L176

Reply:
yes in theory,  I suppose